packages/core/src/components/forms/_numeric-input.scss

```diff
@@ -146,6 +146,8 @@ string }> {
 Styleguide components.forms.numeric-input.js.controlled
 */
 
+$numeric-input-button-height: ($pt-button-height / 2) - 1px;
```

Comment:
Actually, before I implement `pt-large` on `NumericInput`, we'll need a better story around `pt-large` support in generic control groups. I'd like to be able to add a `pt-large` class on the control group itself, since enlarging individual elements would never look good. That's a big enough change for a separate PR, I think.